### PR TITLE
cli: Add `nns-name` and `nns-zone` for `container create`

### LINKED ISSUE #37: Remove --name parameter from frostfs-cli
## Problem
The `--name` parameter implies setting a name for created container. However, this does not happen since the procedure was changed and requires setting special attributes with `--attributes` parameter. Thus the `--name` parameter and its description in command line help became misleading.

## Solution
1) Remove the `--name` parameter from command line.
2) Improve the documentation and make the information on setting container's name more accessible.

### PR DESCRIPTION
**Pull request by acid-ant**
Close #37 

Signed-off-by: Anton Nikiforov <an.nikiforov@yadro.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -46,6 +47,8 @@ Changelog for FrostFS Node
 - Actually use `object.put.pool_size_local` and independent pool for local puts (#64).
 
 ### Removed
+- Parameter `name` from command `frostfs-cli container create` (#37)
```
Comment (fyrchik) [inline CHANGELOG.md:50]: We don't need to remove it, it is still valid, just mean a different thing (application-level readable identifier).

File: cmd/frostfs-cli/modules/container/create.go [hunk 2]
```diff
@@ -209,8 +212,11 @@ func parseAttributes(dst *container.Container, attributes []string) error {
 		container.SetCreationTime(dst, time.Now())
 	}
 
-	if containerName != "" {
-		container.SetName(dst, containerName)
+	if containerNnsName != "" {
+		dst.SetAttribute(containerApi.SysAttributeName, containerNnsName)
```
Comment (fyrchik) [inline cmd/frostfs-cli/modules/container/create.go:216]: Can we use `containerSDK.WriteDomain` here?

File: CHANGELOG.md (hunk 1 above)
Comment (acid-ant) [reply CHANGELOG.md]: Oops, reverted.

File: cmd/frostfs-cli/modules/container/create.go (hunk 2 above)
Comment (acid-ant) [reply cmd/frostfs-cli/modules/container/create.go]: Thanks, replaced.

File: cmd/frostfs-cli/modules/container/create.go [hunk 3]
```diff
@@ -213,5 +217,10 @@ func parseAttributes(dst *container.Container, attributes []string) error {
 		container.SetName(dst, containerName)
 	}
 
+	var doamin container.Domain
```
Comment (fyrchik) [inline cmd/frostfs-cli/modules/container/create.go:220]: `s/doamin/domain/`

Comment (acid-ant) [reply cmd/frostfs-cli/modules/container/create.go]: Fixed.

**Review by dstepanov-yadro — APPROVED** [review]

**Review by fyrchik — APPROVED** [review]

**Pull request merged**